csfieldguide/templates/chapters/chapter_section.html

```diff
@@ -46,14 +46,21 @@
 
 {% block right_column_content %}
   {% if chapter_section.translation_available %}
-    <div id="sticky-sidebar" class="sticky d-print-none">
-      <h2 class="mt-2">{% trans "Table of contents" %}</h2>
-      <ul class="nav nav-pills flex-column" id="scrollspy-table-of-contents">
+    <div id="sticky-sidebar" class="sticky-top d-print-none">
+      <h2 class="mt-3">{% trans "Table of contents" %}</h2>
+      <ul class="nav nav-pills flex-column">
         {% for section in chapter.chapter_sections.all %}
           <li class="nav-item">
             <a class="nav-link {% if section.slug == chapter_section.slug %} active text-white{% else %} nav-link-colour{% endif %}" href="{% url 'chapters:chapter_section' chapter.slug section.slug %}">
               {{ section.name }}
             </a>
+            {% if section.slug == chapter_section.slug %}
+              <nav id="section-headings" class="nav nav-pills flex-column my-1 ml-3">
+                {% for heading in headings %}
+                  <a class="nav-link" href="#{{ heading.slug }}">{{ heading.name }}</a>
+                {% endfor %}
```

Comment:
I'm guessing it's one of the scrollspy things that couldn't be implemented, but I'll point out anyway that it's jarring to click a heading in the table of contents, have the page jump to that heading, and not have the heading I clicked on highlighted

Reply:
Yes, Scrollspy does all that. Going to look into it later.

Reply:
Now fixed